P0009/P0009.md

```diff
@@ -1011,6 +1059,70 @@ for supporting heterogeneous memory.
 For a discussion of the idea of accessors and several examples,
 please see (Keryell and Falcou 2016).
 
+### Why do we distinguish between accessors and pointers?
+
+Our design orthogonalizes storage type or allocation handle (the pointer)
+from access properties (the accessor).
+An `mdspan` stores a `pointer`,
+but uses its accessor for element access and for making offset views.
+Just as `atomic_ref` and ordinary pointer dereferencing
+can access the same memory in two different ways,
+two different `mdspan` objects might access the same memory in two different ways.
+
+This explains why the accessor policy uses `a.access(p, i)`
```

Comment:
why mdspan uses a.access(p,i). Also this duplicates information from above. better to merge.